packages/css/error-message.css

```diff
@@ -1,40 +1,40 @@
-@layer fds.errormessage {
-  .errorMessage {
+@layer fds.typography.error-message {
+  .fds-error-message {
     --fdsc-bottom-spacing: var(--fds-spacing-5);
 
     margin: 0;
   }
 
-  .errorMessage.error {
+  .fds-error-message.fds-error-message--error {
```

Comment:
Remove the `.fds-error-message` on the following lines, they are kinda redundant?

Reply:
removed 👍🏻